app/javascript/packs/application.js

```diff
@@ -0,0 +1,29 @@
+// Entry point for the build script in your package.json
+import "@hotwired/turbo-rails"
+import "../controllers"
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { BrowserRouter, Route, Switch } from 'react-router-dom';
+import { Provider } from 'react-redux';
+import Greeting from '../components/Greeting';
+import store from '../redux/store';
+
+const App = () => {
+  return (
+    <Provider store={store}>
+      <BrowserRouter>
+        <Switch>
+        <Route exact path="/" component={Greeting} />
+        </Switch>
+      </BrowserRouter>
```

Comment:
- [ ]  Also, please consider using the correct syntax for `react-router-dom` as `Switch` was removed in v6. 🤓 🔧